src/style/_mixins.scss

```diff
@@ -1,5 +1,14 @@
 @import "./shadow.mixin";
 
+$ssv-breakpoints: (
+	"xsmall": 450px,
+	"small": 767px,
+	"medium": 992px,
+	"large": 1200px,
+	"xlarge": 1500px,
+	"xxlarge": 1800px
+);
```

Comment:
I'd provide the consumer a way to override this. By specifying the default bang `!default`